django-backend/fecfiler/reports/views.py

```diff
@@ -163,6 +163,18 @@ def partial_update(self, request, pk=None):
         response = {"message": "Update function is not offered in this path."}
         return Response(response, status=status.HTTP_405_METHOD_NOT_ALLOWED)
 
+    def list(self, request, *args, **kwargs):
```

Comment:
I think if we're gonna have to make a change to the api, we should instead make the endpoint that we want: an endpoint where, given a date, it returns a single report that covers that date